frontend-react/src/shared/Citation/Citation.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from "classnames";
+
+export interface CitationProps
+    extends React.PropsWithChildren<
+        React.HTMLAttributes<HTMLElement> & CitationItem
+    > {}
+
+export const Citation = ({
+    className,
+    title,
+    quote,
+    author,
+    authorTitle,
+    ...props
+}: CitationProps) => {
+    const classnames = classNames("padding-top-9", className);
```

Comment:
I don't think we should be passing down any css classes, and I don't think they're even being used here? If we have styles / functionality that are toggled on or off, they should be passed down as props.

Reply:
Good catch. I will fix